Resources/ServerInfo/_NF/Guidebook/Shipyard/Aegis.xml

```diff
@@ -0,0 +1,49 @@
+<Document>
+  # DRAGOON-CLASS PATROL SHUTTLE
+<Box>
+  <GuideEntityEmbed Entity="NFWeaponRifleAssaultGestio" Caption="Negotiate"/>
+  <GuideEntityEmbed Entity="NFPosterContrabandFscLogo" Caption="Profit"/>
+  <GuideEntityEmbed Entity="ClothingOuterEVASuitFsa" Caption="Explore"/>
+</Box>
+<Box>
+  <GuideEntityEmbed Entity="ShuttleMapAegis" Caption=""/>
+</Box>
+  [color=#a4885c]Ship Size:[/color] Medium
+
+  [color=#a4885c]Recommended Crew:[/color] 1-3
+
+  [color=#a4885c]Power Gen Type:[/color] Uranium
+
+  # Shuttle Brief
+
+  "The Aegis is an atmophere capable expedition shuttle designed for long duration exploration missions with the comfort and care of the crew as a mainstay priority."
```

Comment:
Typo: "Atmophere" should be "Atmo**s**phere"

I assume "atmosphere" here means it can tolerate a planetary atmosphere, rather than atmospherics etc. etc.
Not sure if you want to tease out that ambiguity